dist/index.html

```diff
@@ -236,49 +218,70 @@ <h3 class="section-subheading text-muted">A part by part breakdown of how we mak
             </div>
         </div>
     </section>
-    <!-- Team-->
-    <section class="page-section bg-light" id="team">
+
+
+
+
+
+
+    <!-- Order section-->
+    <section class="page-section bg-light" id="Order">
```

Comment:
"order" should also be lowercase here to it matches the navbar. (it should be camelCase if it's anymore than a single word)